README.md

```diff
@@ -1,3 +1,30 @@
 # TileDB-SOMA-Test-Data
 
-WIP
+This project stores test and example data for TileDB-SOMA. Test data is stored as
+
+## Available data
+
+No data has been added.
+
+## Add new raw datasets
+
+Raw datasets and large binaries should be added to this repository as releases. This is done because we do not need the version control functionality of git for the test data and GitHub has a larger file size limit on the releases then files in the repo.
+
+To add new test data do the following:
+
+1. Create a new release in the repository using the GitHub interface.
+2. Pick a title for the dataset (for example "Visium v2.0.0 Dataset") and a description of what the data is. The description may include where the data was originally hosted or how it was otherwise generated.
+3. Upload the desired datasets to the release by using the file dialog in the release page (at time of writing in Firefox this is done by clicking "Attach binaries by dropping them here or selecting them." in the new release page).
```

Comment:
Sorry, I'm still confused. For "Upload the desired datasets" -- what _is_ a desired dataset? What should people be attaching? What should they not be attaching? How will they know if they've attached something correct, or something incorrect?

AFAICT this readme presumes that the reader and the writer both already know (and agree on) what the release artifacts are.

Reply:
Can you maybe show some examples of what a 'desired dataset' might look like? And maybe what a `tar ztf` of one of those should look like?

Also, how people can test-drive their "desired dataset" before attaching it?

Reply:
As discussed offline, how to test-drive desired datasets will be added to the TileDB-SOMA side if/when needed.